doc/tools/coqrst/repl/coqtop.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, coqtop_bin=None, color=False, args=None) -> str:
         self.coqtop_bin = coqtop_bin or os.path.join(os.getenv('COQBIN', ""), "coqtop")
         if not pexpect.utils.which(self.coqtop_bin):
             raise ValueError("coqtop binary not found: '{}'".format(self.coqtop_bin))
-        self.args = (args or []) + ["-color", "on"] * color
+        self.args = (args or []) + ["-q", "-color", "on"] * color
```

Comment:
Should probably be something like:

```suggestion
        self.args = (args or []) + ["-q"] + ["-color", "on"] * color
```

or possibly:

```suggestion
        self.args = (args or ["-q"]) + ["-color", "on"] * color
```

because you surely do not want the use of `-q` to depend on whether `color=True`.

cc @cpitclaudel (who, as the author of this code, might have an opinion on which solution is best).

Reply:
Ah thanks, I see I didn't pay enough attention here... I'll wait for @cpitclaudel to make a choice between the suggestions.

Reply:
I would say that the first one is best though because I cannot imagine why one would want to override the `-q` flag.

Reply:
First one looks good :)